src/components/hotels/hotel-item/hotel-item.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'hotel-item',
+  templateUrl: './hotel-item.component.html',
+  styleUrls: ['./hotel-item.component.scss'],
+})
+export class HotelItemComponent implements OnInit {
+  @Input() hotel: IHotel = {} as IHotel;
```

Comment:
```suggestion
  @Input() hotel = {} as IHotel;
```
if you use `{} as IHotel` you don`t need to set type